edweiss-app/app/(app)/lectures/slides/index.tsx

```diff
@@ -160,16 +168,42 @@ const LectureScreen: ApplicationRoute = () => {
 
     const ContentView = (widthPercent: string, heightPercent: string) => (
         <TView flexDirection='column' mr={'xl'} style={{ width: widthPercent as DimensionValue, height: heightPercent as DimensionValue }}>
+
+            {/* Transcript selection */}
+            <TView style={{ position: 'absolute', top: 10, right: 10, zIndex: 1 }}>
+                <TTouchableOpacity
+                    backgroundColor='crust'
+                    borderColor='text' p={'sm'} b={1} ml={'sm'} radius={1000}
+                    onPress={() => modalRefTranscriptMode.current?.present()}
+                    testID='st-trans-mode-sel-button'>
+
+                    {transMode === 'original' ? (
+                        <Icon size={'lg'} name='language-outline' color='text'></Icon>
+                    ) : (
+                        <TText size={'lg'}>{transModeIconMap[transMode]}</TText>
```

Comment:
Is it really supposed to be a TText in this part? isn't this doable with another type of component? Like Icon, Avatar or something else that I might not know

Reply:
So first, as you master this code, you indeed have remarks my bit bizzare approach. So the reason I did this, os for  user ergonomic. I wanted to signal to the user that the `original` transcript can be translated to other languages. For this I have the `lang`  icon for this. Once the user have selected a lang, I wanted to showcase that's it's the translation and not the original. To this extent, I put a language emoji. Unfortunatly, it do not fit into an `Icon`, so I needed to put it into a `TText`, with the same propert as the `Icon` in order to stay consistent.

Reply:
Okay thank you for the explanation